### ISSUE TITLE
Unable to define a section in .edgerc 

### ISSUE THREAD

**Issue by hamai8CC** [akamai/cli-cps#17]
I am running akamai cps --version
**akamai cps 0.1.0**

When I have an .edgerc file with multiple sections and try to pass my "cps" section, I get the following:
**akamai cps setup --section cps**
Unknown error occurred trying to read edgerc file (/Users/{myLDAPLogin}.edgerc)

When I rename that .edgerc file to .edgercALL and create an .edgerc file with only one section (cps), it runs fine.

**Comment by vreddhi** [comment]
Looks like your original `.edgerc` file is corrupted. You may add another section manually in the new file and confirm. Let me know whether that is the root cause.

**Comment by vreddhi** [comment]
@hamai8CC Were you able to get this working? Can we close this issue?

**Comment by hamai8CC** [comment]
Yes, this can be closed.  I did as you suggested, where there were 2 credentials listed, was able to pass a specified section and it worked fine.  Thank-you.
